recipe/meta.yaml

```diff
@@ -34,6 +34,8 @@ requirements:
     # compatible with all Python versions. Thus as long as the recipe supports
     # python_min<311, have to require exceptiongroup
     - exceptiongroup
+  run_constrained:
+    - zarr <3
```

Comment:
@jdblischak this should get repodata-patched into older releases I think!

Reply:
@flying-sheep Thanks for the heads up! Done in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/944